python/tvm/contrib/cc.py

```diff
@@ -231,6 +233,20 @@ def _linux_compile(output, objects, options, compile_cmd, compile_shared=False):
     else:
         if compile_shared or output.endswith(".so") or output.endswith(".dylib"):
             cmd += ["--shared"]
+
+        compute_version = "".join(
+            get_target_compute_version(Target.current(allow_none=True)).split(".")
+        )
+        cmd += ["-gencode", f"arch=compute_{compute_version},code=sm_{compute_version}"]
+        cmd += ["-O3"]
+        cmd += ["-std=c++17"]
+        cmd += ["-Xcompiler=-fPIC"]
+        cmd += ["-Xcompiler=-fno-strict-aliasing"]
+        cmd += ["-t 24"]
+        cmd += ["-DCUTLASS_ENABLE_TENSOR_CORE_MMA=1"]
+        cmd += ["-Xcompiler=-Wconversion"]
+        cmd += ["--use_fast_math"]
```

Comment:
doesn't make sense we enable fast math by default. We should control it using one option and should set it false by default.